frontend/utilities/permissions/permissions.tests.ts

```
@@ -0,0 +1,110 @@
+
+  it("allows a team admin of all the user's fleets to manage that user", () => {
+    const target = createMockUser({
+      global_role: null,
+      teams: [
+        { id: 1, name: "Team 1", role: "observer" },
+        { id: 2, name: "Team 2", role: "admin" },
+      ],
+    });
+    expect(
+      permissions.isAdminForAllUserTeams(teamAdminTeam1And2, target)
+    ).toBe(true);
```

ESLint

[error] Replace `⏎······permissions.isAdminForAllUserTeams(teamAdminTeam1And2,·target)⏎····).toBe(true` with `permissions.isAdminForAllUserTeams(teamAdminTeam1And2,·target)).toBe(⏎······true⏎····`

(prettier/prettier)